buildSrc/src/main/kotlin/ai.errorprone-conventions.gradle.kts

```diff
@@ -59,9 +59,6 @@ tasks {
         disable("UnnecessaryDefaultInEnumSwitch")
         disable("InconsistentOverloads")
 
-        // consider enabling this after removing azure-monitor-exporter from this repo
-        disable("CanIgnoreReturnValueSuggester")
```

Comment:
it seems this is still needed.

Reply:
that's ok to keep it (and remove the comment)